src/features/grading/GradingTypes.ts

```diff
@@ -9,6 +11,24 @@ import {
 } from '../../commons/assessment/AssessmentTypes';
 import { Notification } from '../../commons/notificationBadge/NotificationBadgeTypes';
 
+export enum ColumnFields {
+  assessmentName = 'assessmentName',
+  assessmentType = 'assessmentType',
+  studentName = 'studentName',
+  studentUsername = 'studentUsername',
+  groupName = 'groupName',
+  progressStatus = 'progressStatus',
+  gradingStatus = 'gradingStatus',
+  xp = 'xp',
+  actionsIndex = 'actionsIndex'
+}
```

Comment:
This might result in coupling; but just to confirm since I'm not too familiar: why must this be a separate definition instead of using TS `keyof`?

Reply:
I did use the string as values in some of the areas in the table, which ```type``` doesn't allow the use of it (or not that I know of at least)